.github/workflows/docker-publish.yml

```
@@ -21,7 +21,23 @@
+    outputs:
+        tag: ${{steps.vars.outputs.tag}}
+        commit: ${{steps.vars.outputs.commit}}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          submodules: true
+      - name: Set output
+        id: vars
+        run:
+            echo "tag=${GITHUB_REF_NAME}" >> $GITHUB_OUTPUT;
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)